src/runtime/process-io.ts

```
@@ -0,0 +1,748 @@
+// =============================================================================
+// CONSTANTS
+// =============================================================================
+
+/** Default maximum response line length: 100MB */
+const DEFAULT_MAX_LINE_LENGTH = 100 * 1024 * 1024;
+
+/** Maximum stderr bytes to retain for diagnostics: 8KB */
+const MAX_STDERR_BYTES = 8 * 1024;
+
+/** Regex for ANSI escape sequences */
+const ANSI_ESCAPE_RE = /\u001b\[[0-9;]*[A-Za-z]/g;
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)